next.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   env: {
-    apiBase: 'https://api.rcvr.app/',
+    apiBase: 'http://172.16.0.90:3000/',
```

Comment:
Obacht, I don't think we want to merge this 😅 

```suggestion
    apiBase: 'https://api.rcvr.app/',
```